src/app/components/carousel/carousel.component.scss

```diff
@@ -26,21 +26,21 @@
       }
     }
 
-    &:hover svg {
-      filter: drop-shadow(0px 4px 8px #000000);
-      opacity: 0.6;
-      transition: opacity 0.5s;
+    // &:hover svg {
```

Comment:
remove commented out code ?

Reply:
I don't know if we might want to re-enable this in the future?

Reply:
We can add it then, I think.